jujugui/static/gui/src/app/init.js

```diff
@@ -526,56 +536,25 @@ class GUIApp {
     });
     state.register([
       ['*', this._ensureControllerConnection.bind(this)],
-      ['*', this.authorizeCookieUse.bind(this)],
       ['*', this.checkUserCredentials.bind(this)],
-      ['*', this._renderComponents.bind(this)],
+      ['*', this._renderApp.bind(this)],
```

Comment:
Instead of dispatching to each render method, just render/update state on the main app component.